src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/Report.kt

```diff
@@ -8,33 +8,57 @@ import java.util.UUID
 
 @Schema(description = "Incident report")
 data class Report(
+  @Schema(description = "The internal ID of this report", required = true)
   val id: UUID,
+  @Schema(description = "The human-readable identifier of this report", required = true)
   val incidentNumber: String,
+  @Schema(description = "Incident report type", required = true)
   val type: Type,
+  @Schema(description = "When the incident took place", required = true, example = "2024-04-29T12:34:56.789012")
   val incidentDateAndTime: LocalDateTime,
+  @Schema(description = "The NOMIS id of the prison where incident took place", required = true, example = "MDI")
   val prisonId: String,
+  @Schema(description = "Brief title describing the incident", required = true)
   val title: String,
+  @Schema(description = "Longer summary of the incident", required = true)
   val description: String,
+  @Schema(description = "Event linking multiple incident reports together", required = true)
   val event: Event,
 
+  @Schema(description = "Username of person who created the incident report", required = true)
   val reportedBy: String,
+  @Schema(description = "When the incident report was created", required = true, example = "2024-04-29T12:34:56.789012")
   val reportedDate: LocalDateTime,
+  @Schema(description = "The current status of this report", required = true)
   val status: Status,
+  @Schema(description = "Optional user who this report is currently assigned to", required = false, defaultValue = "null")
   val assignedTo: String? = null,
 
+  @Schema(description = "The question-response pairs that make up this report", required = true)
   val questions: List<Question> = emptyList(),
+  @Schema(description = "Prior versions of this report, created when the report type changes", required = true)
```

Comment:
I know this is not there yet but would a new version be created when anything changes? not just the type no?

Reply:
apparently only when the type is changed; to match nomis